dataline-singer/src/main/resources/json/SingerMessage.json

```diff
@@ -43,8 +44,12 @@
         "type": "string"
       }
     },
+    "version": {
```

Comment:
this isn't part of the singer spec? would it make sense to do what you did in catalog and just allow additional fields? my intuition is that we only explicitly add the field here is if it something that both source and destination need to know about.